src/index.js

```diff
@@ -1,4 +1,4 @@
-import { cloneElement, h, Component } from 'preact';
+import { cloneElement, h, Component, toChildArray } from 'preact';
```

Comment:
Maybe it's worth switching to `createElement` instead of `h`?
https://github.com/developit/preact/blob/master/src/index.js#L2

Reply:
was thinking the same :-)

Reply:
Doesn't matter. `h` is just an alias to `createElement` and both work exactly the same way.